cmd/chihaya/config.go

```diff
@@ -21,15 +21,27 @@ type hookConfig struct {
 	Config interface{} `yaml:"config"`
 }
 
+type hookConfigs []hookConfig
+
+// Names returns all hook names listed in the configuration.
+func (hookCfgs hookConfigs) Names() (hookNames []string) {
+	hookNames = make([]string, 0)
```

Comment:
Oh I think I need to assign to the allocated spaces instead of append now:

![screen shot 2017-06-02 at 8 51 49 pm](https://cloud.githubusercontent.com/assets/4852029/26749665/592ee0dc-47d5-11e7-9c4a-131b8bada78b.png)